src/pyhf/tensor/tensorflow_backend.py

```diff
@@ -44,6 +44,14 @@ def clip(self, tensor_in, min_value, max_value):
             max_value = tf.reduce_max(tensor_in)
         return tf.clip_by_value(tensor_in, min_value, max_value)
 
+    def conditional(self, bool_condition, true_callable, false_callable):
```

Comment:
As this is introducing new functions to `tensorlib` I'll make this a separate PR, but I'll demo and protype them here before I do so.